NetSuiteProject/Exercise.js

```diff
@@ -2,22 +2,65 @@
 function add(x,y){
 	return x + y;
 };
+
+function addition(x,y){
+	if(typeof add (x,y) === "string"){
+	alert("This function only accepts Integers")
+	}
+	else {return x + y;}
+}
+
+
 //subtract
 function subtract(x, y){
 	return (x - y);
 };
+
+function subtraction(x, y){
+	if(isNaN(subtract(x,y))  === true){
+	alert("This function only accepts Integers");
+	}
+	else {return (x - y);}
+}
+
+
 //Multiplication
 function multiply(x, y){
 	return (x * y);
 };
+
+function multiplication(x,y){
+	if(isNaN(multiply(x,y)) === true ){
+		alert("This function only accepts Integers");
+	}
+	else {return (x * y);}
+}
+
+
 //Division
 function divide (x, y){
 	return (x / y);
 };
+
+function division(x, y){
+	if(isNaN(divide(x,y)) === true){
+		alert("This function only accepts Integers");
```

Comment:
What about non-dialog feedback systems? If you try to use your module on the server side, for instance, as it would for instance in a NetSuite User Event script, then there is no user interface, and thus the `alert` etc methods will not be defined or helpful.

What are some other ways your code can send feedback to the developers using them?